driver-core/src/test/unit/com/mongodb/internal/connection/CommandMessageTest.java

```diff
@@ -51,7 +51,7 @@ void encodeShouldThrowTimeoutExceptionWhenTimeoutContextIsCalled() {
         //given
         CommandMessage commandMessage = new CommandMessage(NAMESPACE, COMMAND, FIELD_NAME_VALIDATOR, ReadPreference.primary(),
                 MessageSettings.builder()
-                        .maxWireVersion(THREE_DOT_SIX_WIRE_VERSION)
+                        .maxWireVersion(LATEST_WIRE_VERSION)
```

Comment:
I did not get the same change - my (naive) merge resulted in, I believe, no changes to this file. Could you explain these changes and why they were needed to make this merge work?

Reply:
We removed support for 3.6 in this PR: https://github.com/mongodb/mongo-java-driver/pull/1375.

However, it should be FOUR_DOT_ZERO_WIRE_VERSION. I have corrected it to FOUR_DOT_ZERO_WIRE_VERSION. Thanks!